src/libotr.mk

```diff
@@ -9,7 +9,7 @@ $(PKG)_CHECKSUM := 8b3b182424251067a952fb4e6c7b95a21e644fbb27fbd5f8af2b2ed87ca41
 $(PKG)_SUBDIR   := $(PKG)-$($(PKG)_VERSION)
 $(PKG)_FILE     := $(PKG)-$($(PKG)_VERSION).tar.gz
 $(PKG)_URL      := https://otr.cypherpunks.ca/$($(PKG)_FILE)
-$(PKG)_DEPS     := gcc libgcrypt
+$(PKG)_DEPS     := gcc libgcrypt libgpg_error
```

Comment:
I believe that all dependencies should be specified explicitly. Consider what will happen, if libgcrypt stops depending on libgpg_error, but libotr still depends on libgpg_error. Probably we will just update libgcrypt without adding libgpg_error as a libotr dependency, if that happens.